jtec-core/src/main/java/edu/tum/sse/jtec/instrumentation/testevent/TestEventInterceptorUtility.java

```diff
@@ -117,35 +107,46 @@ public static void incrementRunCount() {
     }
 
     public static void testStarted(final Description description) {
-        if (currentTestCase.equals(getTestCaseName(description))) {
+        final String testName = getTestCaseName(description);
+        if (currentTestCase.equals(testName)) {
             return;
         }
-        currentTestCase = getTestCaseName(description);
-        currentTestSuite = getTestSuiteName(description);
+        currentTestCase = testName;
+
+        final String suiteName = getTestSuiteName(description);
+        if (!currentTestSuite.equals(suiteName)) {
+            if (!currentTestSuite.equals("")) {
```

Comment:
Is there an .empty() for strings in java?

Reply:
.isEmpty() exists it seems. Replace all occurrences.